tests/saw/spec/handshake/handshake.saw

```diff
@@ -87,23 +91,23 @@ let prove_handshake_io_lowlevel = do {
 
 let prove_state_machine = do {
     print "Checking proof that the TLS1.2 RFC simulates our Cryptol s2n spec";
-    prove_print abc {{ tls12rfcSimulatesS2N `{16} }};
+    prove_print (w4_unint_z3 []) {{ tls12rfcSimulatesS2N `{16} }};
```

Comment:
Might be worth comments on prove_state_machine and prove_cork_uncork that they use w4 only because of enable_lax_loads_and_stores. What happens if a different solver is used? Will it produce some known error you can note?

Reply:
I've added disclaimers to each of the functions in this file that contain proofs warning that only What4-based proof tactics should be used. I've also included a typical error message that will arise if another form of proof tactic is used.